src/compiler/python_generator.cc

```diff
@@ -760,6 +760,32 @@ PythonGrpcGenerator::PythonGrpcGenerator(const GeneratorConfiguration& config)
 
 PythonGrpcGenerator::~PythonGrpcGenerator() {}
 
+static bool GenerateGrpc(GeneratorContext* context, PrivateGenerator& generator,
+                         grpc::string file_name, bool generate_in_pb2_grpc) {
+  bool success = false;
```

Comment:
This may be a C++ thing I don't understand, but can we get away without assigning a value to this field at this point? The value assigned here is never read from the field before the field is reassigned, right?